app/controllers/internal/configs_controller.rb

```diff
@@ -26,19 +26,17 @@ def create
 
   def config_params
     allowed_params = %i[
-      default_site_email mascot_user_id
-      campaign_hero_html_variant_name campaign_sidebar_enabled campaign_featured_tags
-      campaign_sidebar_image
+      default_site_email mascot_user_id campaign_hero_html_variant_name
+      campaign_sidebar_enabled campaign_featured_tags campaign_sidebar_image
       main_social_image favicon_url logo_svg logo_png primary_sticker_image_url
-      mascot_image_url mascot_image_description
-      rate_limit_follow_count_daily
+      mascot_image_url mascot_image_description rate_limit_follow_count_daily
       ga_view_id ga_fetch_rate community_description authentication_providers
-      community_member_description tagline
-      mailchimp_newsletter_id mailchimp_sustaining_members_id
-      mailchimp_tag_moderators_id mailchimp_community_moderators_id
-      periodic_email_digest_max periodic_email_digest_min suggested_tags
-      rate_limit_comment_creation rate_limit_published_article_creation
-      rate_limit_image_upload rate_limit_email_recipient sidebar_tags
+      community_member_description tagline mailchimp_newsletter_id
```

Comment:
I personally don't think we should change the style of this method. I preferred it when the sections were grouped by their related sections